quiz.js

```diff
@@ -36,25 +52,63 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
+db.westeros.update({
+    name: 'Stark'
+}, {
+    $push: { people: 'Ned Stark' }
+});
+
+db.westeros.update({
+    name: 'Stark'
+}, {
+    $push: { people: 'Arya Stark' }
+});
+
+db.westeros.update({
+    name: 'Stark'
+}, {
+    $push: { people: 'Sansa Stark' }
+});
+
+db.westeros.update({
+    name: 'Targaryen'
+}, {
+    $push: { people: 'Viserys Targaryen' }
+});
+
+db.westeros.update({
+    name: 'Targaryen'
+}, {
+    $push: { people: 'Daenerys Targaryen' }
+});
+
+db.westeros.update({
+    name: 'Jon Arryn'
+}, {
+    $push: { people: 'Jon Arryn' }
+});
 
+// your code ends here
 
 
 
 
-// your code ends here
-
 // Question 3
 //
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
 
+db.westeros.update({
+    name: "Arryn"
+}, {
+  $unset: { motto: 1 }
+});
 
+// your code ends here
```

Comment:
Correct! Though, for clarity, it might have been better to pick an empty string as the value to pair with motto - that might make it more clear that you're deleting the property.